components/board/genericlinux/data.go

```diff
@@ -92,76 +75,47 @@ func readIntFile(filePath string) (int, error) {
 	return int(resultInt64), err
 }
 
-func getGpioChipDefs(pinDefs []PinDefinition) (map[string]gpioChipData, error) {
-	gpioChipsInfo := map[string]gpioChipData{}
-	sysfsPrefixes := []string{"/sys/devices/", "/sys/devices/platform/", "/sys/devices/platform/bus@100000/"}
+// getGpioChipDefs returns of map of chip ngpio# to the corresponding gpio chip name.
+func getGpioChipDefs(pinDefs []PinDefinition) (map[int]string, error) {
+	gpioChipsInfo := map[int]string{}
```

Comment:
This isn't used any more until the loop starting on line 110. I'd move it down there, to limit its scope, so readers don't need to think about it for the next 30 lines.